### PR TITLE
proxyd: Unwrap single RPC batches

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

File: proxyd/backend.go [hunk 1]
```diff
@@ -349,7 +349,17 @@ func (b *Backend) setOffline() {
 }
 
 func (b *Backend) doForward(ctx context.Context, rpcReqs []*RPCReq, isBatch bool) ([]*RPCRes, error) {
-	body := mustMarshalJSON(rpcReqs)
+	isSingleElementBatch := len(rpcReqs) == 0
```
Comment (Inphi) [inline proxyd/backend.go:352]: shouldn't this be `len(rpcReqs) == 1`?

Comment (mslipper) [reply proxyd/backend.go:352]: You are absolutely right. Nice catch, this was sloppy on my part.

Comment (mslipper) [reply proxyd/backend.go]: Fixed.

**Review by Inphi — APPROVED** [review]

**Review by tynes — APPROVED** [review]

**Comment by mslipper** [comment]
@Inphi I re-requested review since I had to modify an existing test to handle this case.

**Review by Inphi — APPROVED** [review]
LGTM!

**Comment by mslipper** [comment]
Thanks!

**Pull request merged**